recipes/busybox/busybox-1.24.1/defconfig

```diff
@@ -451,13 +451,13 @@ CONFIG_FEATURE_FIND_LINKS=y
 CONFIG_HALT=y
```

Comment:
"Don't enable runit related commands when USE_busybox_init," -> "Don't enable runit related commands when USE_busybox_init is set,"